spec/unit_tests/stanford/storage_object_validator_spec.rb

```diff
@@ -8,14 +8,19 @@
     let(:verification_array) { storage_obj_validator1.validation_errors }
 
     it 'returns errors from calling the superclass validation_errors' do
-      expect(verification_array.count).to eq(2)
+      erroneous_druid = 'xx000xx0000'
+      erroneous_druid_path = 'spec/fixtures/bad_root01/bad_moab_storage_trunk/xx/000/xx/0000/xx000xx0000'
+      erroneous_object = Moab::StorageObject.new(erroneous_druid, erroneous_druid_path)
+      erroneous_object_validator = described_class.new(erroneous_object)
+      error_list = erroneous_object_validator.validation_errors
+      expect(error_list.count).to eq(9)
     end
     it 'returns validation error codes when there is file path for druid to directory validation' do
```

Comment:
I know this was already here, but:  this description doesn't read right.

I like the new error (detecting when there are no version directories) but we do need STANFORD::SOV to have a test to show D2D checking fails.

I think the "missing all version directories" should also be a spec, and it should live in Moab::SOV specs